Errors on closing editor

**Describe the bug**
When the editor is closed with the close button, errors appear in the console

```
index.js:513 Error when trying to update AttributeConnection(<Window - window for interactives editor>.close --> <InteractivesEditor - interactives editor>.abandon) with value [object Object]:
TypeError: Cannot read property 'attributeConnections' of undefined
TypeError: Cannot read property 'attributeConnections' of undefined
    at Window.disconnect (index.js:41)
    at GlobalTimeline.abandon (index.js:368)
    at eval (morph.js:1213)
    at Array.forEach (<anonymous>)
    at TabContentContainer.abandon (morph.js:1213)
    at eval (morph.js:1213)
    at Array.forEach (<anonymous>)
    at Tabs.abandon (morph.js:1213)
    at eval (morph.js:1213)
    at Array.forEach (<anonymous>)
```

They should not appear. Perhaps something is wrong

--- a/editor.js
+++ b/editor.js
@@ -183,11 +183,11 @@ export class InteractivesEditor extends Morph {
     this.interactive.remove();
     this.inspector.deselect();
     this.preview.showEmptyPreviewPlaceholder();
-    this.setProperty('interactive', undefined);
   }
 
   reset () {
     this.clearInteractive();
+    this.setProperty('interactive', undefined);
     this.tabContainer.visible = false;
   }
 

